MANIFEST.in

```diff
@@ -1,12 +1,12 @@
+include requirements*.txt
```

Comment:
I think this is still missing a `recursive-include elephant *.py`, otherwise many .py files seem missing when doing a `pip install .`

Reply:
`recursive-include elephant *.py` should not be there (by default, all python files are included). I put this line to avoid some strange behavior with TestPypi only.
Please check it yourself.

1. remove `recursive-include elephant *.py` from manifest
2. `python setup.py sdist`
3. tar --list --file dist/elephant-0.6.2.tar.gz

In particular, make sure there are .py files in `spade_src` and `current_source_density_src` folders.